app/modules/sightings/models.py

```diff
@@ -431,7 +431,8 @@ def get_debug_sighting_json(self):
         return self._augment_edm_json(edm_response)
 
     def get_augmented_sighting_json(self):
-        response = current_app.edm.get_dict('sighting.data_complete', self.guid)
+        if is_extension_enabled('edm'):
```

Comment:
This was in the users resources code before it called edm, not sure why so as this will be removing the users resources way though, copied the check to here.